sdk/synapse/azure-analytics-synapse-artifacts/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0-beta.9 (2022-03-08)
 
 ### Features Added
 
+- Added `ScriptActivity`.
+- Added `TeamDeskLinkedService`, `QuickbaseLinkedService`, `SmartsheetLinkedService`, `ZendeskLinkedService`.
```

Comment:
nit: adding some more details may help here. Not sure why we have `ScriptActivity` in a separate line and the rest of the `*Service` types in one line.

Reply:
Sure. Class on 2 lines is added to different polymorphic base class. I will update the changelog.